### ISSUE TITLE
Privacy manifest file - no nutrition label shown at App Store (SwiftClient 1.5.0)

### ISSUE THREAD

**Issue by kmatzka** [TelemetryDeck/SwiftClient#132]
I am using SwiftClient 1.5.0. According to the docs a privacy manifest file (“PrivacyInfo.xcprivacy”) should be copied to the project and as a result, the privacy nutrition label on the AppStore should be configured accordingly.

But the nutrition label does not show any data tracking information, it states “no data collected”.

The package file references the PrivacyInfo file:

.target(
      name: “TelemetryClient”,
      resources: [.copy(“PrivacyInfo.xcprivacy”)]

And there is a PrivacyInfo.xcprivacy file in the TelemetryClient folder of the package.

It just seems to not get copied into the project.

**Comment by kmatzka** [comment]
BTW, if I want to change the default privacy manifest file, as I am using a userID instead of the deviceID, do I just add an additional manifest file to my project? Does this override the TelemetryClient provided one?

**Comment by kmatzka** [comment]
Update:
I successfully checked that the TelemetryClients PrivacyInfo.xcprivacy file is part of the apps archive by creating a Privacy Report in Xcode Organizer. I guess all is fine with TelemetryClients privacy manifest.

What I misunderstood:
An archive's privacy manifest is NOT automatically creating the relating privacy nutrition labels on the App Store. Needs to be done manually. The Organizers Privacy Report helps to understand the combined privacy data of one's app including package dependencies (that should include their own PrivacyInfo.xcprivacy file).

**Comment by mbuchetics** [comment]
Yes, I also think that the documentation is misleading here ("If you’re using the Swift SDK 1.5.0 or above, you can safely ignore the rest of this document."). Xcode will aggregate the privacy manifests into a combined report but it is not automatically updating the information on App Store Connect.

**Comment by Jeehut** [comment]
You both are totally right, including an `.xcprivacy` file doesn't mean Xcode will fill out the privacy labels on Connect. It's just for the generated report. @mbuchetics I'm happy to adjust the docs accordingly if they still state what you quoted, but I can't find that exact text in the docs. Can you please tell me where you found the misleading docs so I can fix them?

Thank you in advance!
